changedetectionio/templates/settings.html

```diff
@@ -83,7 +83,18 @@
                     </span>
                     </fieldset>
 
-
+                    <fieldset class="pure-group">
+                      {{ render_field(form.global_subtractive_selectors, rows=5, placeholder="header
+footer
+nav
+.stockticker") }}
+                      <span class="pure-form-message-inline">
+                      <br/>
+                      Ignore HTML tags or matches to CSS selectors.
+                      <br/>
+                      Add one tag or CSS selector per line to ignore multiple parts of the HTML.
```

Comment:
"Add multiple elements or CSS selectors per line to ignore multiple parts of the HTML." is better